TWLight/users/models.py

```diff
@@ -148,6 +149,37 @@ def delete_my_library_cache(self):
         """
         return cache.delete(self.my_library_cache_key)
 
+    @property
+    def talk_page_i18n(self):
+        """
+        This property will return the URL of the relevant Wikipedia Library talk page
+        in the user's preferred language
+        """
+        # Adding en wiki because it will not be included in the languages response
+        wiki_twl_pages = {
+            "en": "https://en.wikipedia.org/wiki/Wikipedia:The_Wikipedia_Library"
+        }
+        page = "Wikipedia:The_Wikipedia_Library"
+        url = "https://en.wikipedia.org/w/rest.php/v1/page/" + page + "/links/language"
+        headers = {
+            "User-Agent": "MediaWiki REST API docs examples/0.1 (https://www.mediawiki.org/wiki/API_talk:REST_API)"
+        }
+
+        response = requests.get(url, headers=headers)
```

Comment:
this is adding a synchronous backend request to wikipedia as a dependency for every cold load of my library. I suggest using an fixed include list and only returning values that we know are good (eg. active language page or en).

Reply:
I have added a fixed list. Is this what you meant? If so, I would add this list into a JSON file and load its contents. I would also create a small cron job that updates this JSON file with a Wikipedia request a couple of times a year or yearly.

Reply:
That's good! The code and names are already in the db for user language, so it seems like those should both use the same source of truth for that mapping if feasible (whether they both lookup the db or the json file). But otherwise, yeah, I think just shipping this in a json file like `locale/language-data.json` makes sense!

Reply:
Done!